package.json

```
@@ -39,7 +39,7 @@
   "dependencies": {
     "xmldom": "~0.1.19",
     "concat-stream": "~1.5.1",
-    "minimist": "1.2.0"
+    "minimist": "1.2.3"
```

```suggestion
    "minimist": "~1.2.3"
```

Should work & allow future minors